src/constants/WalletAndCurrencyConstants.ts

```diff
@@ -138,6 +139,7 @@ interface SpecialCurrencyInfo {
   stakeActions?: { [stakeActionKey: string]: string }
   stakeLockPeriod?: number
   stakeMaxApy?: number
+  maxSpendTargets?: number
```

Comment:
I was aware of this need and fully agree, but can we do this in a new PR. Trying to keep this PR relatively contained as it's already a beast. Also the rebase to fix this might also take a couple hours.